Source/GameView/Repositories/GameAPIRepo.swift

```diff
@@ -23,6 +23,21 @@ struct JsonGame: Codable {
   let name: String
 }
 
+fileprivate struct gameDataBundleKeys {
+  static let game: String = "game"
```

Comment:
Much bigger improvement, also consider in future if you'll be using this key anywhere else in the project and consider making a keys struct if necessary where you can group everything.